global-cli/index.js

```diff
@@ -87,7 +87,10 @@ function createApp(name, verbose, version) {
     version: '0.0.1',
     private: true,
   };
-  fs.writeFileSync(path.join(root, 'package.json'), JSON.stringify(packageJson));
+  fs.writeFileSync(
+    path.join(root, 'package.json'),
+    JSON.stringify(packageJson, null, 2)
```

Comment:
this doesn't really matter so much but the initial version of package.json wasn't formatted nicely.